components/automate-ui/src/app/modules/user/user-details/user-details.component.html

```diff
@@ -17,62 +9,38 @@
       <chef-page-header>
         <div class="detail-row">
           <div class="name-column">
-            <ng-container *ngIf="!updatingUser">
-              <div class="display3">{{ user?.name }}</div>
-            </ng-container>
-            <ng-container *ngIf="updatingUser">
-              <form [formGroup]="editForm">
-                <chef-form-field class="display3">
-                  <label>
-                    <span class="label">Full Name <span aria-hidden="true">*</span></span>
-                    <input chefInput formControlName="fullName"/>
-                  </label>
-                  <chef-error
-                    *ngIf="(editForm.get('fullName').hasError('required') || editForm.get('fullName').hasError('pattern')) && editForm.get('fullName').dirty">
-                    Full Name is required.
-                  </chef-error>
-                </chef-form-field>
-              </form>
-            </ng-container>
-            <div class="container-big-header inline-container">
-              <span class="text"> {{ user?.id }}</span>
-            </div>
-          </div>
-          <div class="button-column">
-            <ng-container *ngIf="!updatingUser">
-              <app-authorized [allOf]="[['/auth/users/{username}', 'put', user?.id], ['/iam/v2/users/{username}', 'put', user?.id],
-                ['/auth/users/{username}', 'get', user?.id], ['/iam/v2/users/{username}', 'get', user?.id]]">
-                <chef-button secondary class="edit-button" (click)="setUpdatingUser(true)">
-                  <chef-icon>mode_edit</chef-icon>
-                  <span>Edit</span>
-                </chef-button>
-              </app-authorized>
-              <app-authorized [allOf]="[['/auth/users/{username}', 'delete', user?.id], ['/iam/v2/users/{username}', 'delete', user?.id],
-                ['/auth/users/{username}', 'get', user?.id], ['/iam/v2/users/{username}', 'get', user?.id]]">
-                <chef-button *ngIf="isAdminView" secondary caution class="delete-button"
-                  (click)="openDeleteConfirmationModal()">
-                  <chef-icon>delete</chef-icon>
-                  <span>Delete user</span>
-                </chef-button>
-              </app-authorized>
-            </ng-container>
-            <ng-container *ngIf="updatingUser">
-              <chef-button primary class="save-button" [disabled]="!editForm.valid || !editForm.dirty"
-                (click)="saveFullName()">
-                <span>Save</span>
-              </chef-button>
-              <chef-button tertiary (click)="setUpdatingUser(false)">Cancel</chef-button>
-            </ng-container>
+            <div class="header-user-name" *ngIf="!isAdminView">Profile: {{ user?.name }}</div>
```

Comment:
```suggestion
            <div class="header-user-name" *ngIf="!isAdminView">{{ user?.name }}</div>
```

Reply:
Oh, I thought you wanted "Profile" in the name from here https://app.zeplin.io/project/5ba808703eb54372d860c737/screen/5d362bb7a9e3ddb29ccb5700